### PR TITLE
Fix: update test expected results

### LINKED ISSUE #14554: Fix broken RB tests
## Test 1
 testAllReferencesToClassDoMore – MacOSX64.Refactoring.Tests.Core.RBNamespaceTest
Error
Got #(#testCanRemoveAReferencedClassNotHavingReferenceInTheModel #testCanRemoveUnreferencedClassUsingLimitedEnvironmentButReferencedElsewhere #testRemoveAClassAndTheirSubclass #testRemovingAnEmptyNonLeafClassReparent #testPreconditionNotEmptyClass #testRemoveAClassAndTheirSubclass2UsingAlimitedEnvironment #testRemovingAnEmptyNonLeafClassIsAllowed #testCanRemoveUnreferencedClass #testCanRemoveSuperclass #testCanRemoveReferencedClass) instead of #(#testCanRemoveUnreferencedClass #testCanRemoveReferencedClass #testCanRemoveSuperclass #testRemovingAnEmptyNonLeafClassIsAllowed #testCanRemoveUnreferencedClassUsingLimitedEnvironmentButReferencedElsewhere #testRemoveAClassAndTheirSubclass #testRemovingAnEmptyNonLeafClassReparent #testRemoveAClassAndTheirSubclass2UsingAlimitedEnvironment #testPreconditionNotEmptyClass #testCanRemoveAReferencedClassNotHavingReferenceInTheModel).
Stacktrace
TestFailure
Got #(#testCanRemoveAReferencedClassNotHavingReferenceInTheModel #testCanRemoveUnreferencedClassUsingLimitedEnvironmentButReferencedElsewhere #testRemoveAClassAndTheirSubclass #testRemovingAnEmptyNonLeafClassReparent #testPreconditionNotEmptyClass #testRemoveAClassAndTheirSubclass2UsingAlimitedEnvironment #testRemovingAnEmptyNonLeafClassIsAllowed #testCanRemoveUnreferencedClass #testCanRemoveSuperclass #testCanRemoveReferencedClass) instead of #(#testCanRemoveUnreferencedClass #testCanRemoveReferencedClass #testCanRemoveSuperclass #testRemovingAnEmptyNonLeafClassIsAllowed #testCanRemoveUnreferencedClassUsingLimitedEnvironmentButReferencedElsewhere #testRemoveAClassAndTheirSubclass #testRemovingAnEmptyNonLeafClassReparent #testRemoveAClassAndTheirSubclass2UsingAlimitedEnvironment #testPreconditionNotEmptyClass #testCanRemoveAReferencedClassNotHavingReferenceInTheModel).

## Test2
Attempt to read undeclared variable RBRemoveClassParametrizedTest
Stacktrace
UndeclaredVariableRead
Attempt to read undeclared variable RBRemoveClassParametrizedTest
UndeclaredVariable>>runtimeUndeclaredRead
RBRemoveClassRefactoringTest>>testPreconditionHasNoReferencesWithClassesWithReferencesBetweenThem
RBRemoveClassRefactoringTest(TestCase)>>performTest

## Test 3

osx-64 / Tests-osx-64 / testNoUnusedClassVariablesLeft – MacOSX64.ReleaseTests.CleanCode.NoUnusedVariablesLeftTest
3s
Error
the following classes have unused class variables and should be cleaned: an OrderedCollection(MyClassNonEmptyLeafUnused)

## Test 4
NoUnusedVariablesLeftTest
7s
Error
the following classes have unused instance variables and should be cleaned: an OrderedCollection(MyClassNonEmptyLeafUnused)

## Test 5
Categorization.ProperMethodCategorizationTest
<1s
Error
the following classes have uncategorized methods: an OrderedCollection(MyClassNonEmptyLeafUnused RBNamespace)

### PR DESCRIPTION
**Pull request by balsa-sarenac**
This PR fixes some test failures that are introduced by previous PRs.

Closes: #14554

### CODE REVIEW

**Review by jecisc — APPROVED** [review]
Thank you!

**Comment by jecisc** [comment]
@balsa-sarenac can you check the CI result to see if this is what you expected?

**Comment by balsa-sarenac** [comment]
@jecisc I'm unable to see the CI results 😞 
Can you perhaps copy/paste it so I can check?

**Comment by jecisc** [comment]
86C800F3-E043-4B8F-B395-8E7108BC2644
EDCD0809-C94B-4E77-B55C-FDCB3CAC72B8
050E3CFD-2606-4B2C-81A4-B8207037BB03

**Comment by Ducasse** [comment]
Oh great I was creating an issue just for that.

**Comment by Ducasse** [comment]
https://github.com/pharo-project/pharo/issues/14554

File: src/Refactoring-Tests-Core/RBNamespaceTest.class.st [hunk 1]
```diff
@@ -146,7 +147,7 @@ RBNamespaceTest >> testClassesReferencingClass [
 	classes := model classesReferencingClass: (model classFor: RBFooDummyLintRuleTest1).
 	self
 		assert: (classes asArray collect: [ :each | each name ]) 
-		equals:  #(#RBRemoveClassRefactoringTest #RBRemoveClassTransformationTest)
+		equals: #(#RBRemoveClassTransformationTest #RBRemoveClassRefactoringTest)
```
Comment (balsa-sarenac) [inline src/Refactoring-Tests-Core/RBNamespaceTest.class.st:150]: Here as well. Look at the comment above ^

**Pull request merged**